src/main/java/io/reactivex/Observable.java

```diff
@@ -13,26 +13,70 @@
 
 package io.reactivex;
 
-import java.util.*;
-import java.util.concurrent.*;
-
 import org.reactivestreams.Publisher;
 
-import io.reactivex.annotations.*;
+import java.util.Collection;
```

Comment:
Please do not unroll star imports.

Reply:
Fixed.

Reply:
@akarnokd Maybe you can include some rule for import to `CONTRIBUTING.md` ?

Reply:
@jordan1997 

> When submitting code, please make every effort to follow existing conventions and style in order to keep the code as readable as possible.

But don't worry, we will warn you if there is a stylistic problem.